BASD-MS-119/Dashboard view

AgustinCarthery opened the pull request:
## Dashboard 

A dashboard was made with the intention of listing the most recent activity of the applicant

- [ ] Display latest positions
- [ ] Display Incoming sessions
- [ ] Display latest applications

Commit:
MS-119: initial dashboard setup

Commit:
MS-119: small table fix

Commit:
MS-119: styles update

Commit:
MS-119: applications widget added

veronabaudracco reviewed (changes requested):
-Postulant should be allowed to cancel or reschedule the session from the dashboard
-Postulant should be allowed to cancel schedule interviews from the dashboard

Commit:
MS-119: redux thunks updated in update methods to refetch

Commit:
MS-119: Home component updated with Cancel option in interviews and sessions

AgustinCarthery commented:
> -Postulant should be allowed to cancel or reschedule the session from the dashboard -Postulant should be allowed to cancel schedule interviews from the dashboard

Fixed, wasn't aware of A/C for the issue, thanks

btw changes had to be made on server in order to methods work, redux was also updated to refetch on FE
